### PR TITLE
Updated spinners

### PR DESCRIPTION
**Pull request by inakijv**
Most important aspect:
- Added one more eZ logo spinner version.

screen shot 2017-03-07 at 12 21 25 pm

### CODE REVIEW

**Comment by inakijv** [comment]
Ping @SylvainGuittard

**Comment by inakijv** [comment]
Reviewed spinners based on feedback :-)
Ping @SylvainGuittard 

screen shot 2017-03-09 at 11 09 46 am

File: ui-guidelines/css/resources-icons.css [hunk 1]
```diff
@@ -69,28 +74,60 @@
...
-.ez-spin-sq {
-    font-size: 2rem;
+.ez-spin-sq-fill {
+    font-size: 2.5rem;
     position: absolute;
-    margin-top: .6rem;
-    margin-left: .6rem;
+    margin-top: 1.75rem;
+    margin-left: 1.75rem;
     background-color: #ffffff;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-icons.css]: `#fff` is better

Comment (inakijv) [inline ui-guidelines/css/resources-icons.css]: Good call. Thanks! :-)
Updated now ping @SylvainGuittard

**Pull request merged**